src/main/java/org/chiclepad/backend/Dao/ChiclePadUserDao.java

```diff
@@ -0,0 +1,157 @@
+package org.chiclepad.backend.Dao;
+
+import org.chiclepad.backend.LocaleUtils;
+import org.chiclepad.backend.entity.ChiclePadUser;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Locale;
+
+public class ChiclePadUserDao {
+
+   // Dara source
+   private JdbcTemplate jdbcTemplate;
+
+   protected ChiclePadUserDao(JdbcTemplate jdbcTemplate) {
+      this.jdbcTemplate = jdbcTemplate;
+   }
+
+   //CREATE
+   public ChiclePadUser create(String email, String password, String salt) throws DuplicateKeyException {
+
+      String sqlInsert = "INSERT INTO chiclepad_user(id,email, password, salt)"
+            + " VALUES(DEFAULT ,?,?,?) RETURNING id ;";
+
+      Object id = null;
+      try {
+         id = jdbcTemplate.queryForObject(sqlInsert, new Object[] { email, password, salt }, Integer.class);
+      } catch (DuplicateKeyException e) {
+         System.out.println("This user already exists.");
+      }
+
+      return id == null ? null : new ChiclePadUser((int) id, email, password, salt);
+   }
+
+   //READ
+   public ChiclePadUser get(int id) {
+      String sqlGet = "SELECT * FROM chiclepad_user"
+            + " LEFT OUTER JOIN chiclepad_user_details ON chiclepad_user_details.user_id = " + id
+            + " WHERE chiclepad_user.id =" + id + ";";
+
+      try {
+         return jdbcTemplate.queryForObject(sqlGet,
+               (RowMapper<ChiclePadUser>) (ResultSet resultSet, int rowNum) -> {
+                  return getChiclePadUser(resultSet);
+               }
+         );
+      } catch (EmptyResultDataAccessException e) {
+         System.out.println("User with this id: " + id + " does not exist.");
```

Comment:
We have a `logger` for sysouts.
There is `System.err.println` for errors.
StackTrace would be useful for fixing the issue.